### ISSUE TITLE
Unknown signature algorithm

### ISSUE THREAD

**Issue by pzb** [lgarron/badssl.com#66]
On the "extremely bad" list is an unknown signature algorithm.  Something that is completely unknown.  See the certificates for https://w3.awstls.com/

What is scary is that some clients accept the certificate!

**Comment by lgarron** [comment]
Are you suggesting this as a potential subdomain?

https://w3.awstls.com/ gives me `ERR_CONNECTION_REFUSED` in Chrome.

**Comment by pzb** [comment]
Yes, I'm suggesting it as a potential subdomain.  And yes, any sane client will reject the connection.

**Comment by lgarron** [comment]
> Yes, I'm suggesting it as a potential subdomain.

Hmm. I'm presuming that it's not possible to get an otherwise "valid" publicly signed cert that has an unknown signature algorithm. Do you know of a way?

**Comment by marumari** [comment]
I mean, we could just take an existing certificate and change the OID.  For this cert, I'm reasonably certain that the only thing that changed is that the OID for the signature algorithm got changed from, presumably:

1.2.840.113549.1.1.1

To:

1.2.840.113549.1.1.99

As such, I think this is just a variation of issue #29.
